stable/etcd-operator/templates/restore-operator-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.deployments.restoreOperator }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.restoreOperator.name }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "etcd-restore-operator.fullname" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: "{{ .Release.Name }}-{{ .Values.component }}"
+spec:
+  ports:
+  - protocol: TCP
+    name: http-etcd-restore-port
+    port: {{ .Values.restoreOperator.port }}
+  selector:
+    app: {{ template "etcd-restore-operator.fullname" . }}
```

Comment:
Once the `app` label is changed as suggested above, you also need to add the `release` label to the selector.